turbopack/crates/turbopack-core/src/reference/mod.rs

```diff
@@ -292,14 +310,12 @@ pub async fn primary_chunkable_referenced_modules(
                 ResolvedVc::try_downcast::<Box<dyn ChunkableModuleReference>>(*reference)
             {
                 if let Some(chunking_type) = &*reference.chunking_type().await? {
-                    let resolved = reference
-                        .resolve_reference()
-                        .resolve()
-                        .await?
-                        .primary_modules()
-                        .owned()
-                        .await?;
-                    return Ok(Some((chunking_type.clone(), resolved)));
+                    let result = reference.resolve_reference().resolve().await?;
+                    let resolved = result.primary_modules().owned().await?;
+
+                    let export = result.await?.export.clone();
+
+                    return Ok(Some((chunking_type.clone(), export, resolved)));
```

Comment:
`ExportUsage` should be stored on the reference itself. As far as I can tell, the resolving inside of `resolve_reference` never actually uses the `ExportUsage` and it's merely passed out again here.
If it's not involved in `resolve_reference`, we'd get more cache just like today.